docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -102,6 +102,9 @@ properties:
   - Hive metastore authentication type. Possible values are `NONE` or
     `KERBEROS`.
   - `NONE`
+* - `hive.metastore.catalog-name`
+  - The term "Hive metastore catalog name" refers to the abstraction concept within Hive, enabling various systems to connect to distinct, independent catalogs stored in the metastore. By default, the catalog name in Hive metastore is set to "hive." When this configuration property is left empty, it defaults to accessing the "hive" catalog.
```

Comment:
`When this configuration property is left empty, it defaults to accessing the "hive" catalog.` -> `When this configuration property is left empty, the default catalog of the Hive metastore will be accessed`.